model_compression_toolkit/exporter/model_wrapper/__init__.py

```diff
@@ -13,8 +13,12 @@
 # limitations under the License.
 # ==============================================================================
 
-from model_compression_toolkit.exporter.model_wrapper.keras.validate_layer import is_keras_layer_exportable
-from model_compression_toolkit.exporter.model_wrapper.keras.builder.fully_quantized_model_builder import get_exportable_keras_model
+from mct_quantizers.common import constants
 
-from model_compression_toolkit.exporter.model_wrapper.pytorch.validate_layer import is_pytorch_layer_exportable
-from model_compression_toolkit.exporter.model_wrapper.pytorch.builder.fully_quantized_model_builder import get_exportable_pytorch_model
+if constants.FOUND_TF:
```

Comment:
All these API functions/classes have protection against missing tf/torch packages in the function/class level (with FOUND_TF or FOUND_TORCH).
So you don't need to add it here as well.
Did you encounter any missing FOUND_TF/FOUND_TORCH in any of the APIs? If so, it would be better to add it there.

Reply:
I had to put them because it would crash otherwise because I don't use TF.

Reply:
> I had to put them because it would crash otherwise because I don't use TF.

Can you detail in which file the error occurs? Can you provide the error stack?
It might be that there is a missing FOUND_TF condition somewhere else in the code.

Reply:
Hi @mikeseven,

Can you please provide the information regarding where the program is failing because of this missing condition so we can validate it and move forward with the PR?

Thanks